testing/testapp/setup.go

```diff
@@ -91,7 +91,7 @@ const (
 var DefaultConsensusParams = &abci.ConsensusParams{
 	Block: &abci.BlockParams{
 		MaxBytes: 200000,
-		MaxGas:   2000000,
+		MaxGas:   -1,
```

Comment:
why set maxgas to -1

Reply:
-1 means no limit so that we can test with many txs in one block.